### ISSUE TITLE
optimize for v8?

### ISSUE THREAD

**Issue by yoshuawuyts** [sethvincent/send-action#3]
There's two ways of calling `send()`:

``` js
/*
* Send an action to the store
*/
send({ type: 'example' value: 'cool' })

/*
* Alternate `send` syntax
*/
send('example', { value: 'cool' })
```

If there's only one way v8 is happier - not sure we should care about this, but figured I'd put it out there (':

**Comment by sethvincent** [comment]
Interesting. How much happier? 😄 

If I were to pick one I'd probably go with `send('example', { value: '' })` only because `send('whatever')` is  compact.

**Comment by sethvincent** [comment]
addressed this in v3.0.0 🎉 ¯\_(ツ)_/¯.

there is now only the `send('actionName', data)` syntax.
